### PR TITLE
Fixes #29901 - Lock default Job templates

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: app/models/template.rb [hunk 1]
```diff
@@ -87,6 +87,7 @@ def import_without_save(text, options = {})
     self.snippet = !!@importing_metadata[:snippet]
     self.default = options[:default] unless options[:default].nil?
     self.description = @importing_metadata[:description]
+    self.locked = options[:locked] unless options[:locked].nil?
     handle_lock_on_import(options)
```
Comment (ares) [inline app/models/template.rb:91]: note that this line already contains the logic, the problem is the argument name

File: app/models/template.rb [hunk 2]
```diff
@@ -87,6 +87,7 @@ def import_without_save(text, options = {})
     self.snippet = !!@importing_metadata[:snippet]
     self.default = options[:default] unless options[:default].nil?
     self.description = @importing_metadata[:description]
+    self.locked = options[:locked] unless options[:locked].nil?
```
Comment (ares) [inline app/models/template.rb:90]: the plugin should use `:lock`, not `:locked`, then it would be handled by the line below I believe

**Review by ares — CHANGES_REQUESTED** [review]
I think this PR can be closed and seeding fixed in all plugins that seed the templates, see the inline comment for more details

**Comment by stejskalleos** [comment]
Plugin updates PR: https://github.com/oamg/foreman_leapp/pull/95 & https://github.com/theforeman/foreman_remote_execution/pull/500

**Pull request closed without merging**